pages/querying/expressions.mdx

```diff
@@ -136,3 +136,59 @@ predicates. In this case, anything that is not true is interpreted as being fals
 
 The [`exists()`](/querying/functions#pattern-functions) function cannot be used with the CASE clause.
 
+## Pattern comprehension
```

Comment:
Not good examples, let's have our own simple examples utilizing pattern comprehension that users can try out.

Reply:
@katarinasupe I don't quite get what is wrong with the examples. Are you referring to examples for Pttern comprehension?